src/routes/task/task.ts

```diff
@@ -1,24 +1,27 @@
 const router = require('express').Router()
-const Model = require('./task/task');
+const Model = require('./taskModel')
 
 router.post('/api/tasks', async(req, res) => {
-    const data = new Model({
-        text: req.body.text,
-        date: new Date().toLocaleString(),
-        description: '',
-        taskStatus: 'todo'
-    })
-    try{
-        const dataToSave = data.save();
+    console.log(req.user)
+    try {
+        const data = new Model({
+            text: req.body.text,
+            date: new Date().toLocaleString(),
+            description: '',
+            taskStatus: 'todo',
+            userID: req.user._id

```

Comment:
jak jest akronim pisany dużymy literami jak np `ID` lub `URL` to używając camelCase piszesz tylko pierwsze litere duża czyli `userId`